Assets/Scripts/Weapons/BehaviourTypes/ArtilleryMono.cs

```diff
@@ -37,12 +35,11 @@ public static GameObject create(ArtilleryAttackBehaviour attackBehaviour, Vector
         
         Vector3 positionXY = MathModule.convertToXY(behaviour.gpPlane, position, behaviour.d);
         Vector3 targetXY = MathModule.convertToXY(behaviour.gpPlane, direction, behaviour.d);
-        if (targetXY.x < positionXY.x) behaviour.dir = -1f;
         Vector3 vertexXY = MathModule.convertToXY(behaviour.gpPlane, vertex, behaviour.d);
 
         behaviour.quadratic = MathModule.calculateQuadratic(new Vector3[3] { positionXY, targetXY, vertexXY });
         
-        behaviour.incrementX = Mathf.Abs(targetXY.x - positionXY.x)/200; 
+        behaviour.incrementX = (targetXY.x - positionXY.x)/300; 

```

Comment:
does this change slow down the projectile?

Reply:
Yes.